crates/autopilot/src/solvable_orders.rs

```diff
@@ -227,15 +228,15 @@ impl SolvableOrdersCache {
         }
 
         let removed = counter.checkpoint("missing_price", &orders);
-        order_events.extend(removed.into_iter().map(|o| (o, OrderEventLabel::Filtered)));
+        filtered_order_events.extend(removed);
 
         let orders = filter_mispriced_limit_orders(orders, &prices, &self.limit_order_price_factor);
         let removed = counter.checkpoint("out_of_market", &orders);
-        order_events.extend(removed.into_iter().map(|o| (o, OrderEventLabel::Filtered)));
+        filtered_order_events.extend(removed);
```

Comment:
Oh wow, the code is quite lavish with the reasons for which it excludes orders. Would love to see all of those go away (and be done in the driver eventually). This has nothing to with this PR of course.